src/docs/database-and-api-management/opensource-database-technologies.md

```diff
@@ -29,7 +29,7 @@ Postgres is a [relational database](https://insightsoftware.com/blog/whats-the-d
 
 [Patroni](https://github.com/bcgov/patroni-postgres-container) is the most common highly available solution on the platform. It is very well-supported by the development community in the BC Government. However, it is older and less flexible than the CrunchyDB Operator. You can find a Platform Team supported image for Patroni at `artifacts.developer.gov.bc.ca/bcgov-docker-local/patroni-postgres`.
 
-[CrunchyDB](https://github.com/bcgov/crunchy-postgres) is easier to implement and has many more powerful functions than Patroni. It's also more resilient to failure than Patroni. However, since it is new, there are currently fewer community-created tools and templates for CrunchyDB. The operator includes its own image to ensure the best possible compatibility, so the Platform Team does not provide an image for use with the operator. We recommend that you read through the [official CrunchyDB documentation](https://access.crunchydata.com/documentation/postgres-operator/4.1.2/gettingstarted/) to get a full understanding of the features offered by the operator.
+[CrunchyDB](https://github.com/bcgov/crunchy-postgres) is built on Patrino, making it easier to implement and has many more powerful functions than Patroni. It's also more resilient to failure than Patroni. However, since it is new, there are currently fewer community-created tools and templates for CrunchyDB. The operator includes its own image to ensure the best possible compatibility, so the Platform Team does not provide an image for use with the operator. We recommend that you read through the [official CrunchyDB documentation](https://access.crunchydata.com/documentation/postgres-operator/4.1.2/gettingstarted/) to get a full understanding of the features offered by the operator.
```

Comment:
```suggestion
[CrunchyDB](https://github.com/bcgov/crunchy-postgres) is built on Patroni, making it easier to implement and has many more powerful functions than Patroni. It's also more resilient to failure than Patroni. However, since it is new, there are currently fewer community-created tools and templates for CrunchyDB. The operator includes its own image to ensure the best possible compatibility, so the Platform Team does not provide an image for use with the operator. We recommend that you read through the [official CrunchyDB documentation](https://access.crunchydata.com/documentation/postgres-operator/4.1.2/gettingstarted/) to get a full understanding of the features offered by the operator.
```